### DISCUSSION TITLE
Various errors

### DISCUSSION THREAD

**Discussion by MatthK** [SecureAI-Tools/SecureAI-Tools#18] [Q&A]
Came across your Reddit post and since I value privacy, I really like this. So I have to try it, and a big thanks already.

After some initial problems, I got it running. However, when I try to go to the settings page again, I get the following error message:

````
Something went wrong!
Something went wrong while trying to load data. Please refresh your page to try again; If the problem still persists, please contact your organization administrator.
````

Then another one. I first used used Firefox to try and logged in. I changed the password, but didn't realize I have to actually download the mistral model. So every question resulted in an error in the log file. When I tried to go into the settings again, I got the same error message as above. Since I just started, I just deleted the two directories `inference` and `web` and started again. 
But now I can't use Firefox anymore, it says:

```
No organizations!
Your account (bruce@wayne-enterprises.com) is not a member of any organizations. Please contact your organization administrator and ask them to add you to the organization or activate your membership.
```

Couple of other questions:

- How do I log out?
- How to change the default user's email?
- How to create more users?
-

**Comment by JayNakrani** [comment]
Hey, thanks for trying this out. How are you running the instance? If you're running it via docker-compose then it should have seeded the db with correct organization and membership. But if you are running it directly through `docker run` or some other way, then it's quite possible that data wasn't seeded properly.

Either way, it'd be good to query the SQLite3 db file and see what type of data is in the Organization and OrgMembership tables. Feel free to DM DB query outputs privately to me on discord at https://discord.gg/YTyPGHcYP9

Now, specifically to your questions:

> How do I log out?

Please visit http://localhost:28669/log-out to log out.


> How to change the default user's email?

We haven't yet built this in the web UI. For now, you can query the SQLite DB file directly via [SQLite cli](https://www.sqlite.org/index.html) to update user's email.

> How to create more users?

Please visit `http://localhost:28669/<org-slug>/users/add` where `<org-slug>` is the `Organization.slug` from DB. If you're using docker-compose set-up then org slug is dash `-` so the URL would be `http://localhost:28669/-/users/add`

We know that navigation today is very simple and lacking in many aspects, as this is the v0 release. We will continue to improve navigation :)

**Comment by JayNakrani** [comment]
I'm converting this to a discussion and if we have any bugs or feature requests come out of that discussion, then we'll file them separately as GH issues. Feel free to continue on the discussion thread :)

**Comment by MatthK** [comment]
I do run it via docker compose, as suggested on the main page here.

After having logged out and then logged in again, it does work again on Firefox too. So I guess it was just that it still remembered the previous login.
